kiwidrive/xbox.py

```diff
@@ -34,8 +34,8 @@ def analog_rot(self):
                self.joystick.getRawButton(XboxController.BUTTON_BUMP_LEFT)
 
     def analog_winch(self):
-        return self.joystick.getRawJoy(XboxController.JOY_TRIG_R) -\
-               self.joystick.getRawJoy(XboxController.JOY_TRIG_L)
+        return self.joystick.getRawButton(XboxController.JOY_TRIG_R) -\
+               self.joystick.getRawButton(XboxController.JOY_TRIG_L)
```

Comment:
``JOY_TRIG_{R,L}`` is a Joy, not a Button. Did you want to put in ``BUTTON_BUMP_{R,L}``?